core/src/test/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -144,10 +144,11 @@ private void mkdir(String path) throws IOException {
   }
 
   public void start() throws IOException {
-    mTachyonHome =
-        File.createTempFile("Tachyon", "").getAbsoluteFile() + "U" + System.currentTimeMillis();
+    mTachyonHome = File.createTempFile("Tachyon",
+        "U" + System.currentTimeMillis()).getAbsolutePath();
```

Comment:
Will the previous style not work for this case? ie.
mTachyonHome =
  File.createTempFile ...

I think it is cleaner if possible.